daemons/attrd/pacemaker-attrd.h

```diff
@@ -31,7 +31,7 @@
  * --------  ---------  -------------------
  *     1       1.1.11   PCMK__ATTRD_CMD_UPDATE (PCMK__XA_ATTR_NAME only),
  *                      PCMK__ATTRD_CMD_PEER_REMOVE, PCMK__ATTRD_CMD_REFRESH,
- *                      PCMK__ATTRD_CMD_FLUSH, PCMK__ATTRD_CMD_SYNC_RESPONSE
+ *                      PCMK__ATTRD_CMD_SYNC_RESPONSE
```

Comment:
bump the attrd protocol for a change to accepted commands. here, replace this with `"flush"` and add `"flush" support dropped` for the new protocol number